tldw_Server_API/app/core/AuthNZ/database.py

```
@@ -181,6 +232,68 @@ def _apply_single_user_fallback(url: str, auth_mode: Optional[str] = None) -> st
+        if missing_api_key_cols:
+            raise RuntimeError(
+                "SQLite api_keys schema missing required columns: "
+                + ", ".join(missing_api_key_cols)
+            )
+
+        scope_default = api_key_table_info["scope"][4]
+        if scope_default is not None:
+            raise RuntimeError(
+                "SQLite api_keys.scope must not define a default; "
+                f"found {scope_default}"
+            )
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)